packages/server-wallet/src/wallet/multi-threaded-wallet/manager.ts

```diff
@@ -64,6 +64,7 @@ export class WorkerManager {
     const resultPromise = new Promise<MultipleChannelOutput>((resolve, reject) =>
       worker.once('message', (response: Either<Error, MultipleChannelOutput>) => {
         this.pool?.release(worker);
+
```

Comment:
I added this whitespace and force-pushed because circle failed to trigger a CI run.